Developers/Writing_Good_Commits.rst

```diff
@@ -139,6 +139,7 @@ More Examples
 -------------
 
```

Comment:
@Adrianzatreanu he did set up his git authorship. Just that his email in git config isn't same as the github profile. That's completely fine IMO and some people don't mind the wrong email (I mix up my github and work email all the time)